### DISCUSSION TITLE
Embed a raw JSON file in Webpack

### DISCUSSION THREAD

**Discussion by koleto** [quasarframework/quasar#16473] [CLI - PWA mode] [answered]
For a webpack based project I need to include options.json file without modifying the name or minifying it so that I can exchange the file options.json on the server directory and, still be able to import the modified options.json in the app. Something like:
`module.exports = {
  loaders: [
    {
      options: /\.json$/,
      loader: 'raw-loader',
    },
  ],
};`

How to include this into `quasar.config.js`?

I try to use 'static' and can use it like: `import options from 'src/static/options.json'`, but the file gets minified and disappears somewhere in the bundle.

**Comment by yusufkandemir** [comment, marked as answer]
`raw-loader` will load the file as a string, so it will be a static value on runtime. So, there will not be any file to exchange it later.

What you want instead is to put it into `/src/public`, then use `fetch` to access it on runtime.

**Reply by koleto** [reply]
Thanks!
